crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -214,12 +214,22 @@ It can greatly improve performance (and readability) in such situations as it pr
         re_tracing::profile_function!();
 
         // For all following lookups, checking indicators is enough, since we know that this is enough to infer visualizability here.
-        let Some(indicated_entities) = ctx
-            .indicated_entities_per_visualizer
-            .get(&LegacyTimeSeriesSystem::identifier())
-        else {
+        let mut indicated_entities = IndicatedEntities::default();
+
+        for indicated in [
+            LegacyTimeSeriesSystem::identifier(),
+            SeriesLineSystem::identifier(),
+            SeriesPointSystem::identifier(),
+        ]
```

Comment:
Hmm if one would only log the new scalar archetype, then this wouldn't spawn a space view. But then again we also don't know what visualizer to activate right now, so I figure this is correct? All a bit confusing with the Legacy one around right now

Reply:
This actually works because the SeriesLineSystem and SeriesPointSystem both accept Scalar as an indicator.